### DISCUSSION TITLE
[질문] 개발자도구 네트워크탭 활용법

### DISCUSSION THREAD

**Discussion by moolmin** [100-hours-a-week/discussions#51] [개발] [answered]
<h2>Q. 질문</h2>
그동안 네트워크탭에서 Status Code만 보고 넘겼는데, 네트워크탭을 더 야무지게 활용할 수 있는 방법이 있을까요?

<h2>A. 제푸의 답변</h2>
1. Filter 텍스트 박스 활용
특정 method만 필터링할 수 있다. <br>
ex) method: GET <br>
<img width="600" alt="Screenshot 2024-08-22 at 4 51 42 PM" src="https://github.com/user-attachments/assets/7c105b25-2249-4df2-80a7-e3f6a53ca9f8">
<br><br><br>
2. Initiator 활용<br>
Network > Initiator 탭에서  어떤 파일에서 요청한 것인지 진입점을 확인할 수 있다.
<img width="600" alt="Screenshot 2024-08-22 at 4 53 38 PM" src="https://github.com/user-attachments/assets/8a76d41b-752e-46a2-a613-e289a2a9451f">

**Comment by ej31** [comment, marked as answer]
개발자도구는 신이에요.. 사람들이 잘 몰라서 너무 슬픕니다 😬
